mypyc/common.py

```diff
@@ -1,27 +1,31 @@
-PREFIX = 'CPyPy_'  # Python wrappers
-NATIVE_PREFIX = 'CPyDef_'  # Native functions etc.
-DUNDER_PREFIX = 'CPyDunder_'  # Wrappers for exposing dunder methods to the API
-REG_PREFIX = 'cpy_r_'  # Registers
-STATIC_PREFIX = 'CPyStatic_'  # Static variables (for literals etc.)
-TYPE_PREFIX = 'CPyType_'  # Type object struct
-ATTR_PREFIX = '_'  # Attributes
+MYPY = False
+if MYPY:
```

Comment:
Is the `if MYPY` here needed? Final should just be present in `typing_extensions`